File: web/src/model/victron_controller/dashboard/from_0_1_0_command_set-forecast-disagreement-strategy.ts
// @ts-nocheck
import {SetForecastDisagreementStrategy as v0_1_0_SetForecastDisagreementStrategy} from './v0_1_0/Command'
import {SetForecastDisagreementStrategy as dashboard_SetForecastDisagreementStrategy} from './Command'

export function convert__command__set_forecast_disagreement_strategy__from__0_1_0(from: v0_1_0_SetForecastDisagreementStrategy): dashboard_SetForecastDisagreementStrategy {
    return new dashboard_SetForecastDisagreementStrategy (
        JSON.parse(JSON.stringify(from.value))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_command_set_forecast_disagreement_strategy.rs


pub fn convert__command__set_forecast_disagreement_strategy__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::command::SetForecastDisagreementStrategy) -> crate::victron_controller::dashboard::command::SetForecastDisagreementStrategy {
    crate::victron_controller::dashboard::command::SetForecastDisagreementStrategy {
        value: serde_json::from_value(serde_json::to_value(&from.value).unwrap()).unwrap(),
    }
}